WizardComparison.md

```
@@ -607,29 +651,30 @@ if (data.TryGet<YourStepModel>(out var yourStep))
-
+| Aspect | Inline Fun Wizard | Questionary Wizard | Person Wizard |
+|--------|-------------------|-------------------|--------------|
+| **Registration** | Inline ViewModel factories | Static Registry | DI-based Definition |
+| **Service Injection** | No service injection | No service injection | Full support |
+| **Step Factories** | `Func<IWizardStepViewModel>` | `Func<IWizardStep>` | `Func<IServiceProvider, IWizardStep>` |
+| **Custom Logic** | Minimal reusable steps | Reusable library steps | Custom step classes |
+| **Complexity** | Very low (starter flow) | Low (simple forms) | High (business rules) |
+| **Edit Mode** | No | No | Bidirectional mapper |
+| **Dynamic Visibility** | Not used | Not used | Supported |
+| **Live Validation** | Standard only | Standard only | Custom logic |
+| **Best For** | Tiny starter flows | Simple forms | Complex workflows |
```

markdownlint-cli2 (0.21.0)

[warning] Tables should be surrounded by blank lines

(MD058, blanks-around-tables)